src/js/components/streaming-api-handler.tsx

```diff
@@ -300,14 +300,15 @@ class StreamingAPIHandler extends React.Component<Props, void> {
     if (accessToken && teamId) {
       // Logged in, inside the app. Deployment View was not the landing page
       url = `${streamingAPIUrl}/${teamId}?token=${encodeURIComponent(accessToken)}`;
```

Comment:
Could you once again go through why the `teamId` is *only* available if this was *not* the landing page? Wouldn't it simplify the code if there were some 'authorization' code block that is run always, regardless of the route?